test/apps/test_init_mt.c

```diff
@@ -50,6 +61,10 @@ int main(int argc, char **argv)
 
 #pragma omp barrier
 
-    printf("finished %d threads\n", count);
-    return 0;
+    gettimeofday(&finish, NULL);
+    duration = finish.tv_sec - start.tv_sec;
```

Comment:
IMO, no need to print duration, or print %d.%d (including usec)

Reply:
> IMO, no need to print duration, or print %d.%d (including usec)

ok, removed duration